modelservice/src/main/java/com/expedia/adaptivealerting/modelservice/providers/graphite/Tags.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2018-2019 Expedia Group, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expedia.adaptivealerting.modelservice.providers.graphite;
+
+import lombok.Data;
+
+@Data
+public class Tags {
+
+    private String name;
+}
```

Comment:
`Tags` should not be a class with a hardcoded tag name, since the actual tag names are arbitrary. Here is a real example:

```
"tags": {
  "lob": "flights",
  "what": "bookings",
  "m_account": [redacted],
  "name": "booking",
  "region": "expedia-all",
  "m_region": [redacted],
  "m_application": [redacted]
}
```

Instead `GraphiteResult` should contain a map called `tags` and allow any dynamic (i.e. not hardcoded) set of tags.

Reply:
yeah. good point. fixed it.

Reply:
Wouldn't we just remove the `Tags` class?